docs/reference/verifier.md

```diff
@@ -50,11 +50,12 @@ The verifier configuration is [TBD] YAML/JSON  with the following properties
 
 ##### Plugin Configuration objects
 
-The following are the keys used to describe configuration of  individual plugins.
+The following are the keys used to describe configuration of individual plugins.
 
 | Property | Type | IsRequired | Description |
 | -------- | -------- | -------- | --------- |
-| name     | string     | true     |The name of the plugin that should match with plugin binary on disk. Must not contain characters disallowed in file paths for the system (e.g. / or \) |
+| name     | string     | true     |The name of the plugin that should match with plugin binary on disk. Must not contain characters disallowed in file paths for the system (e.g. / or \). `name` can be customized when `type` has been specified. |
```

Comment:
Maybe we should mention that when `type` is not provided, `name` would behave like `type`.

Reply:
The binary naming is also confusing to me. If only the name is specified then the binary name must match but otherwsie the binary must match the `type` field right?